_includes/tutorials/finding-distinct/flinksql/markup/test/create-resource-create-clicks-table.sql.template.adoc

```diff
@@ -0,0 +1,4 @@
+Create the following file at `src/test/resources/create-clicks-table.sql.template`. Again, note the `KAFKA_PORT` placeholder since this table uses the Upsert Kafka connector and hence must be able to communicate with Kafka and Schema Registry.
```

Comment:
suggest removing "Again" since this is the first time talking about the placeholder

remove "since this table uses the Upsert Kafka connector and hence must be able to communicate with Kafka and Schema Registry" since the clicks table just uses the `kafka` connector